server/routes/class.ts

```diff
@@ -0,0 +1,98 @@
+import express, { Request, Response } from 'express'
+import { Class } from '../db/models/class'
+import mongoose from 'mongoose'
+
+const router = express.Router()
+
+router.get("/:id", async (req: Request, res: Response) => {
+    //TODO: Check if user has access to class
+    const classId = req.params.id
```

Comment:
this classId can be removed, using 'req.params.id' directly on the following line